scripts/build/Jenkins/Jenkinsfile

```diff
@@ -220,7 +268,42 @@ def GetCrashArtifactDir() {
     }
 }
 
-def CheckoutBootstrapScripts(String branchName) {
+def GetRemoteConfig(repositoryName, repositoryUrl) {
+    def theRemoteConfig = [
+        name: repositoryName,
+        url: repositoryUrl,
+        credentialsId: scm.userRemoteConfigs.credentialsId[0]
+    ]
+    return theRemoteConfig
+}
+
+def GetEngineRemoteConfig() {
+    return GetRemoteConfig(ENGINE_REPOSITORY_NAME, ENGINE_URL)
+}
+
+def GetExtrasRemoteConfig() {
+    return GetRemoteConfig(EXTRAS_REPOSITORY_NAME, EXTRAS_URL)
+}
+
+def UnInstallGitLFS() {
+    withCredentials([usernamePassword(credentialsId: "${env.GITHUB_USER}", passwordVariable: 'accesstoken', usernameVariable: 'username')]) {
+        PlatformSh('git lfs uninstall', 'Git LFS Uninstall')
+    }
+}
+
+def InstallAndPullGitLFS() {
+    if(fileExists('.lfsconfig')) {
+        // Add lfs url to the .lfsconfig file (deleted after run)
+        def lfsURL = sh(script: "git config --get -f .lfsconfig lfs.url", returnStdout: true).toString().trim().replaceAll('https://','')
+        withCredentials([usernamePassword(credentialsId: "${env.GITHUB_USER}", passwordVariable: 'accesstoken', usernameVariable: 'username')]) {
+            PlatformSh("git config -f .lfsconfig lfs.url https://${username}:${accesstoken}@${lfsURL}", 'Set credentials', false)
```

Comment:
I added this thinking this may have been the source of an error that it was most likely not. I will remove it and test it again to make sure.

Reply:
hmmm. Well i removed it and i got a failure:

Reply:
I put it back in and resolves the failure.

Reply:
I looked at the Jenkinsfile for that run, and I saw that the `withCredentials()` was removed.

But you'll also need to remove the following line since it's no longer required. It's also looking for the `accesstoken` that is no longer defined causing the error. 

`PlatformSh("git config -f .lfsconfig lfs.url https://${username}:${accesstoken}@${lfsURL}", 'Set credentials', false)`

I tested it in the latest run in the `o3de-extras-AR` branch and git lfs pull runs successfully.

Reply:
Your right, I missed that. Removing it should work.